extern/open62541/prepare_open62541.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
poor old windows - but OK: I guess this is a 'maintainer level' script is only used to update the amalgamated code for later versions.